tgui/public/tgui.html

```diff
@@ -646,7 +646,7 @@
 </marquee>
 <div id="FatalError__stack" class="FatalError__stack"></div>
 <div class="FatalError__footer">
-Nanotrasen (c) 2525-2562
+Nanotrasen (c) 2525-2563
```

Comment:
```suggestion
Nanotrasen (c) 2525-<script>document.write(new Date().getFullYear() + 540)</script>
```
The funny thing is that we can!

Reply:
we intentionally don't do this because we can't afford any fuck ups

Reply:
> we intentionally don't do this because we can't afford any fuck ups

IE8 supports Date, the worst thing that can happen is someone has their year something stupid and its off for them only

Reply:
or maybe im crazy and we just ignored the last people who said that we should make this a script

Reply:
i'm not going to add this to my PR as I think this should be done as its own PR so we can accurately judge the merits of making it a script (don't have to merge a PR every year) versus fucking up the screen that tells you that the UI is yorked (or yorking the UI itself)

Reply:
This file is already modified by the dm side tgui code before being sent to clients. we have a 3rd option!

Reply:
I'd actually prefer that option!